examples/with-next-js/public/~partytown/partytown-atomics.js

```diff
@@ -0,0 +1,2 @@
+/* Partytown 0.7.4 - MIT builder.io */
```

Comment:
Actually, should any of the `~partytown` directory be committed? It's built as part of the build step right? Otherwise we'll see updates anytime we update partytown?

Reply:
The docs are not super clear on this (searching sourcegraph, I see this both ways). I think you're correct, though, we don't need to commit the ~partytown folder.